src/main.ts

```diff
@@ -1,52 +1,49 @@
 class DrawingApp {
   private canvas: HTMLCanvasElement;
   private context: CanvasRenderingContext2D;
-  private paint: boolean;
   private img: HTMLImageElement;
 
   private clickX: number[] = [];
   private clickY: number[] = [];
-  private clickDrag: boolean[] = [];
+
+  private colonyCount = 0;
+  private colonyCountDisplay: HTMLElement;
 
   constructor() {
     const canvas = document.getElementById('canvas') as HTMLCanvasElement;
     const context = canvas.getContext('2d');
     context.lineCap = 'round';
     context.lineJoin = 'round';
-    context.strokeStyle = 'black';
-    context.lineWidth = 1;
+    context.strokeStyle = 'red';
```

Comment:
```suggestion
    context.fillStyle = 'red';
```

Reply:
or to be semi transparent:
```suggestion
    context.fillStyle = 'rgba(255, 0, 0, 0.5)';
```